db/migrate/20210629104700_populate_participant_profiles.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class PopulateParticipantProfiles < ActiveRecord::Migration[6.1]
+  class EarlyCareerTeacherProfile < ApplicationRecord
+  end
+
+  class MentorProfile < ApplicationRecord
+  end
+
+  class ParticipantProfile < ApplicationRecord
+    self.inheritance_column = nil
+  end
+
+  def up
+    MentorProfile.find_each do |ect_profile|
+      ParticipantProfile.create!(
+        id: ect_profile.id,
+        type: "ParticipantProfile::Mentor",
+        user_id: ect_profile.user_id,
+        school_id: ect_profile.school_id,
+        cohort_id: ect_profile.cohort_id,
```

Comment:
Mentor profiles can theoretically have `nil` cohorts. I don't think there are any in production that do, but to keep our sanity, can we fall back to `Cohort.current.id`?